src/thunderbird_accounts/mail/templates/admin_stalwart_view.html

```diff
@@ -29,5 +29,43 @@ <h1>Stalwart Principals</h1>
     {% endfor %}
     </tbody>
   </table>
+  <div>
+    <br/>
+    <br/>
+    <br/>
+    <br/>
+    {% csrf_token %}
+    <div class="submit-row">
+      <a class="deletelink" id="purge" href="#">Purge individuals</a>
+    </div>
+    <pre id="output"></pre>
+  </div>
 </section>
+<script>
+  // Allows up to clean up the test data before the proper oidc schema has been finallized.
+  const btn = document.getElementById('purge');
```

Comment:
I'll need to clean up users on stage, and maybe prod if prod has any test data lying around.

Reply:
we'll then remove this purge call (and the purge function)